test/e2e/plain-v0_provisioner_test.go

```diff
@@ -472,6 +475,9 @@ var _ = Describe("plain-v0 provisioner bundleinstance", func() {
 
 			err = c.Create(ctx, biDuplicate)
 			Expect(err).To(BeNil(), "failed to create duplicate bundle instance")
+			Eventually(func() error {
+				return c.Get(ctx, client.ObjectKeyFromObject(biDuplicate), &rukpakv1alpha1.BundleInstance{})
+			}).Should(Succeed(), "failed to validate duplicate bundle instance was created in time")
```

Comment:
nit: do we need to worry about `biDuplicate` being created on time via this Eventually? It's really only that `biOriginal` is established on-cluster before `biDuplicate` that seems important to the context of the test.

Reply:
Yeah that's a good call out. I think that you're right and we only care about the original being present.